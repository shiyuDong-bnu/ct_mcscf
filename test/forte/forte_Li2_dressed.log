2025-03-06:11:41:13,949 INFO     [psi4.driver.task_planner:285] PLANNING Atomic:  keywords={'D_CONVERGENCE': 1e-08, 'E_CONVERGENCE': 1e-10, 'MAXITER': 300, 'REFERENCE': 'RHF', 'SCF_TYPE': 'PK', 'function_kwargs': {}}
2025-03-06:11:41:13,951 INFO     [psi4.driver.driver:523] Compute energy(): method=forte, basis=cc-pvdz, molecule=CO, nre=2.381297448015
