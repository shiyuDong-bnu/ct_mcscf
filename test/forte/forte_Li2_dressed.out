
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.9.1 release

                         Git: Rev {HEAD} f53cdd7 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, M. H. Lechner, A. Jiang, S. Behnle, A. G. Heide,
    M. F. Herbst, and D. L. Poole

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Thursday, 06 March 2025 11:41AM

    Process ID: 339947
    Host:       debianlab
    PSIDATADIR: /home/sydong/install_tutorial/psi4/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
import forte

molecule CO{
  0 1
  Li  
  Li  1 2.0
  symmetry c1
}

set {
  basis                 cc-pvdz
  reference             rhf
  scf_type              pk
  maxiter               300
  e_convergence         10
  d_convergence         8
#  docc                  [5,0,1,1]
}

set forte {
  job_type              mcscf_two_step
  active_space_solver                fci
  restricted_docc       [2]
  active                [4,]
  int_type              fcidump
  fcidump_file          DRESSED.DUMP 
  e_convergence         8  # energy convergence of the FCI iterations
  r_convergence         8  # residual convergence of the FCI iterations
  casscf_e_convergence  8  # energy convergence of the MCSCF iterations
  casscf_g_convergence  6  # gradient convergence of the MCSCF iterations
  casscf_micro_maxiter  4  # do at least 4 micro iterations per macro iteration
}

Eforte = energy('forte')
--------------------------------------------------------------------------

Scratch directory: /home/sydong/scratch/psi4/

  Forte
  ----------------------------------------------------------------------------
  A suite of quantum chemistry methods for strongly correlated electrons

    git branch: HEAD - git commit: 259d7b80

  Developed by:
    Francesco A. Evangelista, Chenyang Li, Kevin P. Hannon,
    Jeffrey B. Schriber, Tianyuan Zhang, Chenxi Cai,
    Nan He, Nicholas Stair, Shuhe Wang, Renke Huang
  ----------------------------------------------------------------------------

Traceback (most recent call last):
  File "/home/sydong/install_tutorial/psi4/bin/psi4", line 387, in <module>
    exec(content)
  File "<string>", line 43, in <module>
  File "/home/sydong/install_tutorial/psi4/lib/psi4/driver/driver.py", line 525, in energy
    wfn = procedures['energy'][lowername](lowername, molecule=molecule, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sydong/work/learnspace/quantum_chemistry/software/forte_install/forte/forte/pymodule.py", line 166, in energy_forte
    data = OptionsFactory(options=kwargs.get("forte_options")).run()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sydong/work/learnspace/quantum_chemistry/software/forte_install/forte/forte/core.py", line 140, in wrapper
    result = func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/sydong/work/learnspace/quantum_chemistry/software/forte_install/forte/forte/modules/module.py", line 49, in run
    data = self._run(data)
           ^^^^^^^^^^^^^^^
  File "/home/sydong/work/learnspace/quantum_chemistry/software/forte_install/forte/forte/modules/options_factory.py", line 35, in _run
    data.options.get_options_from_psi4(psi4_options)

RuntimeError: Deprecated option CASSCF_MICRO_MAXITER was set.
 Replaced by MCSCF_MICRO_MAXITER



Printing out the relevant lines from the Psithon --> Python processed input file:
    core.set_local_option("FORTE", "E_CONVERGENCE", 8)
    core.set_local_option("FORTE", "R_CONVERGENCE", 8)
    core.set_local_option("FORTE", "CASSCF_E_CONVERGENCE", 8)
    core.set_local_option("FORTE", "CASSCF_G_CONVERGENCE", 6)
    core.set_local_option("FORTE", "CASSCF_MICRO_MAXITER", 4)
--> Eforte = energy('forte')

!--------------------------------------------------!
!                                                  !
!  Deprecated option CASSCF_MICRO_MAXITER was set. !
!  Replaced by MCSCF_MICRO_MAXITER                 !
!                                                  !
!--------------------------------------------------!

    Psi4 stopped on: Thursday, 06 March 2025 11:41AM
    Psi4 wall time for execution: 0:00:00.26

*** Psi4 encountered an error. Buy a developer more coffee!
*** Resources and help at github.com/psi4/psi4.
